File: CAPITULO 01/C++/ex16.cpp
#include <stdio.h>
int main()
{
	float sal_min,sal_receber,hora_trabalhada,sal_bruto,imp;
	int horas;
	printf("Informe o numero de horas trabalhadas: ");
	scanf("%d%*c",&horas);
	printf("Digite o valor do salario minimo :");
	scanf("%f%*c",&sal_min);
	hora_trabalhada = sal_min/2;
	sal_bruto = horas*hora_trabalhada;
	imp = sal_bruto*0.3;
	sal_receber = sal_bruto - imp;
	printf("O salario a receber e de = %f",sal_receber);
	getchar();
	return 0;
}

File: CAPITULO 01/Java/ex16.java
import java.util.*;
import java.*;
import java.text.*;
public class ex16{
	public static void main(String[]args){
		int horas;
		double sal_min,sal_receber,hora_tra,sal_bruto,imp;
		
		Scanner dado = new Scanner (System.in);
		DecimalFormat cs = new DecimalFormat("0.00");
		System.out.println("Digote o numero de horas trabalhadas ");
		horas = dado.nextInt();
		System.out.println("Digite o valor do salario minímo :");
		sal_min = dado.nextDouble();
		hora_tra = sal_min /2;
		sal_bruto = horas*hora_tra;
		imp = sal_bruto*0.3;
		sal_receber = sal_bruto - imp;
		System.out.println("Salario total "+sal_receber);
	}
}